ios/FluentUI.Tests/FontTests.swift

```diff
@@ -41,4 +41,20 @@ class FontTests: XCTestCase {
         let otherFont = UIFont.systemFont(ofSize: UIFontMetrics.default.scaledValue(for: fontInfo.size))
         XCTAssertEqual(font.fontDescriptor, otherFont.fontDescriptor)
     }
+
+    func testScalingFontForContentSizeCategory() throws {
+        // `size` and `textStyle` should match under the "Large" setting here:
+        // https://developer.apple.com/design/human-interface-guidelines/foundations/typography/#:~:text=Dynamic-,Type,-sizes%20(iOS)
+        let size = 20.0
```

Comment:
Any chance we can get this info from the textStyle instead? If we're just trying to make sure our fonts scale the same way as the system fonts, seems like we shouldn't hard code this here in case Apple decides to change their mind later and break our tests.

Reply:
We can't get it directly through the `UIFont.TextStyle` itself; we'd have to go through a `UIFont` or `UIFontDescriptor` (or something of the like). We can still do this though.